src/features/query-bar/components/filter/Condition.js

```diff
@@ -109,22 +110,54 @@ const Condition = ({
         return [];
     };
 
+    const validateValue = (values: Array<ConditionValueType>, type: string) => {
+        switch (type) {
+            case NUMBER:
+                return isInt(String(values[0]));
+            case FLOAT:
+                return isFloat(String(values[0]));
+            default:
+                break;
+        }
+        return true;
+    };
+
     const getErrorMessage = () => {
         const { values, columnId } = condition;
         const column = columns && columns.find(c => c.id === columnId);
         const type = column && column.type;
 
         const isValueSet = values.length !== 0;
+
+        let isValueValid = true;
+        if (isValueSet && type) {
```

Comment:
Now that `isValueValid` defaults to `true`, you need to set it to `false` if `isValueSet && type` is false. Alternatively, you can set `isValueValid` default value to `false`